Rakefile

```diff
@@ -92,7 +93,9 @@ namespace :test do
 end
 
 desc 'Lint the code through pylint'
-task 'lint' => 'ci:default:lint'
+task 'lint' => ['ci:default:lint'] do
+  sh 'rubocop --fail-level W'
```

Comment:
Why not create a custom Rake task instead and invoke it here ? (and so delete the `RuboCop::RakeTask.new`)
```
require 'rubocop/rake_task'

desc 'Run RuboCop on the ci directory'
RuboCop::RakeTask.new(:rubocop) do |task|
  task.patterns = ['ci/**/*.rb', 'Gemfile, 'Rakefile']
end
```